src/Ractive/prototype/render.js

```diff
@@ -12,7 +12,9 @@ export default function Ractive$render ( target, anchor ) {
 		if ( others ) others.forEach( teardown );
 
 		// make sure we are the only occupants
-		target.innerHTML = ''; // TODO is this quicker than removeChild? Initial research inconclusive
+		if ( !this.enhance ) {
+			target.innerHTML = ''; // TODO is this quicker than removeChild? Initial research inconclusive
```

Comment:
`remove()` and `empty()` only remove jQuery handlers – if someone was storing their own reference, or had added an event listener of their own, I'm not sure there's anything we can do? (We *do* tidy up stuff added with Ractive in the line just above.)